src/ServiceControl.Contracts/ServiceControl.Contracts.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>net8.0</TargetFramework>
```

```suggestion
    <TargetFramework>netstandard2.0</TargetFramework
```

I'm thinking we should go with Netstandard here